src/deepsparse/server/config.py

```diff
@@ -121,6 +121,13 @@ class ServerConfig(BaseModel):
     )
 
     endpoints: List[EndpointConfig] = Field(description="The models to serve.")
+    loggers: Union[Dict[str, Dict[str, Any]], str, None] = Field(
+        default=None,
+        description=(
+            "Optional dictionary of logger integration names to initialization kwargs "
+            "or path to a yaml config file of the same specification. Default is None."
+        ),
```

Comment:
```suggestion
    logger_config: Dict[str, Dict[str, Any]] = Field(
        default={},
        description=(
            "Optional dictionary of logger integration names to initialization kwargs. "
            "Default is empty dictionary."
        ),
```

Reply:
was planning on empty dict meaning no loggers, `None` meaning use the default logger for the integration

Reply:
Ahh I see so then just the Optional[Dict]? Still think `str` should not be one of the options